### PR TITLE
Add RAUC manuals

### PR DESCRIPTION
**Pull request by landerweit-phytec**
Add RAUC manuals for kirkstone, mickledore and scarthgap.

### CODE REVIEW

**Review by mschwan-phytec — CHANGES_REQUESTED** [review]
Thank you for porting the manual! :blush: Looks good overall!

There are only some minor things below...

File: source/rauc/kirkstone.rst [hunk 1]
```diff
@@ -0,0 +1,1030 @@
...
+.. image:: /rauc/images/rauc-ab-system.png
+
+RAUC BSP Example Setup
+----------------------
+
+The partition layout is defined in the ``/etc/rauc/system.conf`` file. As an
+example, this is what it looks like for i.MX 8M Mini with eMMC flash storage::
+
+   [system]
+   compatible=phyboard-polis-imx8mm-4
```
[the remark this answers is not in the document]
Comment (landerweit-phytec) [reply source/rauc/kirkstone.rst]: Done

File: source/conf.py [hunk 2]
```diff
@@ -303,6 +303,30 @@
         'manual',
         False,
     ),
+    (
+        'rauc/kirkstone',
+        'rauc_kirkstone.tex',
```
[the remark this answers is not in the document]
Comment (landerweit-phytec) [reply source/conf.py]: Done

File: README.rst [hunk 3]
```diff
@@ -27,7 +27,7 @@ dependencies are specified in ``requirements/``::
 Make also sure that your system has following tools installed
 (example for debian based systems)::
 
-   apt install latexmk texlive-xetex
+   apt install latexmk texlive-xetex texlive-latex-extra texlive-lang-chinese
```
[the remark this answers is not in the document]
Comment (landerweit-phytec) [reply README.rst]: Done. New PR: #259

**Review by mschwan-phytec — APPROVED** [review]

**Pull request merged**